corgidrp/pump_trap_calibration.py

```diff
@@ -3128,7 +3128,7 @@ def _el_loc_3(sch1, or1, prob1, sch2, or2, prob2, sch3, or3, prob3,
             range = [[0, max(1, E_vals.max()) + 0.5/bins[0]],
                 [0, max(1e-14, cs_vals.max()) + 1e-14*0.5/bins[1]]])
 
-        nrows, ncols, _ = imaging_area_geometry()
+        nrows, ncols, _ = imaging_area_geom('SCI')
```

Comment:
Actually, trap-pumped frames are usually 'ENG', even though the test data was 'SCI'.  For this, I think you should use `ext_hdr['ARRTYPE]`.

Reply:
good call! done!